internal/constants.go

```diff
@@ -14,6 +14,7 @@
 package internal
 
 const (
+	ApiConfigRoute           = "/api/v1/config"
```

Comment:
In core-data and core-metadata we have `b.HandleFunc("/config", configHandler)`, in core-metadata we have `b.HandleFunc(CONFIGENDPOINT, configHandler)` and now we have `mux.Get(internal.ApiConfigRoute, http.HandlerFunc(replyConfig))`. The same (or worst) happens with `/api/v1/ping`. And it's worst with `/api/v1`. But I think this should be done in another PR.
Anyway, why not put this define in `/pkg/clients/constants`? Should be fine to have all API route defines in the same file.

Reply:
Yes, I know exactly what you mean. I put it in internal in this case because it's only used internally by export-client, but you're right that this can and should be normalized -- albeit in another PR.